scripts/prepare_automatic_backups.py

```diff
@@ -34,6 +34,14 @@
 
 
 def generate_backup_url(cloud_storage_bucket_name, module_class_names):
+    """Genrating backup url of update content.
```

Comment:
Spell check -> Generating

Reply:
resolved it.

Reply:
It hasn't resolved yet. Please review again!